book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -40,7 +40,7 @@ statements to make it through this chapter:
 
 ```rust,ignore
```

Comment:
offtopic: why is it ignore?

Reply:
Dunno

Reply:
Otherwise mdbook would try and compile this block, when really it lacks the context necessary to be treated as a complete program.